_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -411,18 +416,10 @@ These `span` elements do not have a [semantic role][] of link.
 
 #### Inapplicable Example 5
 
-These links have empty [accessible names][accessible name].
-
-```html
-<a href="http://facebook.com"></a> <a href="http://twitter.com"></a>
-```
-
-#### Inapplicable Example 6
-
-These image links have empty [accessible names][accessible name].
+The two HTML `a` elements have empty [accessible names][accessible name].
 
 ```html
-<a href="http://facebook.com"><img src="facebook.jpg"/></a> <a href="http://twitter.com"><img src="twitter.jpg"/></a>
+<a href="http://facebook.com"></a> <a href="http://facebook.com"><img src="facebook.jpg"/></a>
```

Comment:
Neither of these are accessible links. Can you make sure this satisfies the SC?

Reply:
Modified the example so that they are both in the accessibility tree but only one has an accessible name.